apps/masterbots.ai/components/routes/chat/chat-search-input.tsx

```
@@ -1,53 +1,55 @@
     if (chatbot) {
       setSearchPlaceholder(chatbot as string)
     } else if (activeCategory && activeCategory !== previousCategory.current) {
       previousCategory.current = activeCategory
-
       const getCategoryLabel = await getCategory({ categoryId: activeCategory })
-
       setSearchPlaceholder(`${getCategoryLabel.name.toLowerCase()} category`)
     }
   }
 
   React.useEffect(() => {
```

Biome

[error] This hook does not specify all of its dependencies: fetchSearchPlaceholder

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)